packages/blocks/src/store/selectors.js

```diff
@@ -644,6 +644,28 @@ export function hasBlockSupport( state, nameOrType, feature, defaultSupports ) {
 	return !! getBlockSupport( state, nameOrType, feature, defaultSupports );
 }
 
+export function __experimentalHasBlockMetadataSupport(
```

Comment:
Why create a new selector instead of using the general one hasBlockSupport?

Reply:
I was in favor of reusing the existing public API. I don't think this feature is distinct enough to create another method that developers have to discover and learn to use.

Reply:
I'm happy to revisit this. I can modify the existing API to serve the same purpose and then adapt the tests. I can do this in a follow up or here directly if you'd prefer.

Reply:
Sure, feel free to choose what works best.